SettingsList.py

```diff
@@ -3140,8 +3144,26 @@ def __init__(self, name, gui_text, min, max, default, step=1,
 
             'Light Arrow Cutscene': Ganon's Castle Boss Key will
             appear on the Light Arrow Cutscene.
+            
+            'Stones': Ganon's Castle Boss Key will be awarded
+            when reaching the target number of Spiritual Stones.
+            
+            'Medallions': Ganon's Castle Boss Key will be awarded
+            when reaching the target number of Medallions.
+                        
+            'Dungeons': Ganon's Castle Boss Key will be awarded
+            when reaching the target number of Dungeon Rewards.
```

Comment:
This includes the 1 in Link's Pocket, right? Should that be noted here? (Should the minimum be 2?)

Reply:
It does include the 1 in Link's Pocket, same as LACS Condition and Bridge Condition. Minimum of 2 would make sense, but it'd also make sense in those as well.